packages/orchestrator-ui-components/src/components/confirmationDialog/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
I think this is specific for the temporary solution of forms, maybe move to the WFO Forms folder